bot/cogs/utils.py

```diff
@@ -59,6 +67,20 @@ def __init__(self, bot: Bot):
             await ctx.invoke(self.bot.get_command("help"), "pep")
             return
 
+        # Handle PEP 0 directly due it's static constant in PEPs GitHub repo in Python file, not .rst or .txt so it
+        # can't be accessed like other PEPs.
```

Comment:
I find "static constant" to be unclear. What does that mean? I think it'll be less confusing if you just leave it out and say the reason is that the PEP doesn't have an associated rst or txt. Also, "due" was not used correctly here - "because" would have been a better fit.